tests/testthat/test_estimate-promoter-activity.R

```diff
@@ -0,0 +1,41 @@
+context('Promoter Activity and Gene Expression')
+library(proActiv)
+
+promoterAnnotation <- promoterAnnotation.gencode.v19
+filesTophat <- list.files(system.file('/extdata/tophat2', package = 'proActiv'), full.names = TRUE, pattern = 'sample1|sample2')
+fileLabels <- paste0('sample', 1:2)
+
+tophatJunctionCounts <- calculatePromoterReadCounts(promoterAnnotation, filesTophat, fileLabels, 'tophat')
+tophatNormCounts <- normalizePromoterReadCounts(tophatJunctionCounts)
+tophatAbsProActiv <- getAbsolutePromoterActivity(tophatNormCounts, promoterAnnotation)
+tophatGeneExp <- getGeneExpression(tophatAbsProActiv) 
+tophatRelProActiv <- getRelativePromoterActivity(tophatAbsProActiv, tophatGeneExp)
+
+test_that('getAbsolutePromoterActivity returns expected output', {
```

Comment:
why not calling the function that you want to test inside of the test?

Reply:
the tests here all check for return value, and not runtime related output like print statements or expected errors.
since the subsequent function calls depend on the previous function calls for their input, and because the scope of a variable is only limited to inside each `test_that`, calling the functions (`getAbsolutePromoterActivity`, `getGeneExpression` and `getRelativePromoterActivity`) outside saves time by running them just once (but the functions being tested here don't actually take very long to run, just mainly for brevity?)